<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SE_VectorDataTranslationGuiBase</class>
 <widget class="QDialog" name="SE_VectorDataTranslationGuiBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>一致性匹配工具</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_14">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>543</width>
       <height>185</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="QGroupBox_input_data_path">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>待翻译矢量数据目录</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="QLabel_input_data_path">
           <property name="text">
            <string>待翻译的矢量数据输入路径</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="QLineEdit_input_data_path">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>输入shp数据所在目录</string>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="QPushButton_input_data_path_browse">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="QGroupBox_config_file_setting">
        <property name="title">
         <string>输入数据参数配置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QGroupBox" name="QGroupBox_character_set">
           <property name="title">
            <string>输入数据字符编码</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="QLabel_character_set">
              <property name="text">
               <string>字符集编码</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="QComboxBox_character_set">
              <item>
               <property name="text">
                <string>UTF-8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GB2312</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GBK</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>743</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="QGroupBox_config_file">
           <property name="title">
            <string>配置文件信息</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="1">
             <widget class="QPushButton" name="QPushBotton_load_config_file">
              <property name="text">
               <string>加载配置文件</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="QLabel_config_file">
              <property name="text">
               <string>指定翻译字段、翻译后的语种、存储翻译后字段名等</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_log_level">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>日志设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="QLabel_log_level">
           <property name="text">
            <string>日志等级</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="QComboxBox_log_level">
           <item>
            <property name="text">
             <string>消息</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>警告</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>关闭</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>调试</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>详细</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>一般错误</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>致命错误</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>688</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="QLabel_log_saving_path">
           <property name="text">
            <string>日志保存路径</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="QLineEdit_log_saving_path">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="QPushButton_log_browse">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_status">
        <property name="title">
         <string>状态</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0">
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="Button_Start">
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="Button_Close">
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
